v4/export/sql.go

```diff
@@ -120,30 +124,51 @@ func SelectAllFromTable(conf *Config, db *sql.DB, database, table string) (Table
 	}, nil
 }
 
-func buildSelectAllQuery(conf *Config, db *sql.DB, database, table string) (string, error) {
+func buildSelectAllQuery(conf *Config, db *sql.DB, database, table string, where string, orderByClause string) (string, error) {
 	var query strings.Builder
+	var err error
 	query.WriteString("SELECT * FROM ")
 	query.WriteString(database)
 	query.WriteString(".")
 	query.WriteString(table)
+
+	separator := "WHERE"
+	if conf.Where != "" {
+		query.WriteString(" ")
+		query.WriteString(separator)
+		query.WriteString(" ")
+		query.WriteString(conf.Where)
+		separator = "AND"
+	}
+
+	if where != "" {
+		query.WriteString(" ")
+		query.WriteString(separator)
+		query.WriteString(" ")
+		query.WriteString(where)
+	}
+
 	if conf.SortByPk {
-		orderByClause, err := buildOrderByClause(conf, db, database, table)
-		if err != nil {
-			return "", err
+		if orderByClause == "" {
```

Comment:
the logic is tricky, consider what if I set  `orderByClause` = `xxx` and set `conf.SortByPk` = false

Reply:
I suggest computing `where` and `orderByClause` in the outside of this function